lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/model/SimplePageToolDaoImpl.java

```diff
@@ -860,14 +871,18 @@ public boolean update(Object o, List<String>elist, String nowriteerr, boolean re
 			if(!(o instanceof SimplePageLogEntry)) {
```

Comment:
looks like this can use the boolean check above?

Reply:
It's in a different function, isn't it?